decidim-core/app/packs/src/decidim/form_filter.component.test.js

```diff
@@ -267,18 +267,18 @@ describe("FormFilterComponent", () => {
       });
 
       it("does not save the state", () => {
-        spyOn(window.history, "pushState");
+        jest.spyOn(window.history, "pushState");
 
         window.onpopstate({ isTrusted: true, state: scopesPickerState});
 
-        expect(window.history.pushState).not.toHaveBeenCalled();
+        expect(window.history.pushState).toHaveBeenCalledTimes(2);
```

Comment:
The idea of this test is to test that the `pushState` is not called when `window.onpopstate` is called.

This spec does not make much sense as is. What I did at 0b6f37332c105f559621734ac23e1bf9cc53a847 was that I added the same expectation before the `window.onpopstate` call to ensure that the number of times the method was called will not change.

I also noted that the `spyOn` we have at line 270 was unnecessary. Or at least the spec was working exactly the same without that.